docs/documentation/introduction/theming.mdx

```diff
@@ -68,11 +66,11 @@ function CustomButtonExample() {
         },
       },
     },
-  }
+  })
 
   return (
     <ThemeProvider value={theme}>
-      <Button>Custom Button</Button>
+      <Button appearance="none">Custom Button</Button>
```

Comment:
Looks like the only reason this was working without `appearance="none"` before was because of the multiple object spreads - the entire `Button` object was being replaced (including all appearances), this appeared to work correctly for the example but it would also remove the `primary`, `minimal`, etc. appearances + other sizes.

It would've needed a `...defaultTheme.components.Button` before `baseStyle` on line 53, and which would've shown the need for `appearance="none"`